src/components/CytoscapeLayout/graphs/GraphHighlighter.ts

```diff
@@ -0,0 +1,126 @@
+const DIM_CLASS: string = 'mousedim';
+
+export class GraphHighlighter {
```

Comment:
Could this not just be a function with some props passed into it?

Reply:
By `props` you mean regular arguments to the function? or a React Component as defined [here](https://reactjs.org/docs/components-and-props.html#functional-and-class-components)?